app-backend/api/src/main/scala/annotation-project/AnnotationProjectTaskRoutes.scala

```diff
@@ -354,16 +354,90 @@ trait AnnotationProjectTaskRoutes
         ScopedAction(Domain.AnnotationProjects, Action.CreateAnnotation, None),
         user
       ) {
-        authorizeAuthResultAsync {
-          AnnotationProjectDao
-            .authorized(
-              user,
-              ObjectType.AnnotationProject,
-              projectId,
-              ActionType.Annotate
+        authorizeAsync {
+          (for {
+            auth1 <- AnnotationProjectDao
+              .authorized(
+                user,
+                ObjectType.AnnotationProject,
+                projectId,
+                ActionType.Annotate
+              )
+            auth2 <- AnnotationProjectDao
+              .authorized(
+                user,
+                ObjectType.AnnotationProject,
+                projectId,
+                ActionType.Validate
```

Comment:
Hmm, I think based on the discussion the other day and the implementation of the share endpoint, I think Validate should not be a super-power action type right? Like, this endpoint should be limited to just checking Annotate action type